File: app/src/test/java/no/nav/sosialhjelp/soknad/web/config/SoknadinnsendingLocalConfig.java
package no.nav.sosialhjelp.soknad.web.config;

import org.mockito.Mockito;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import no.nav.sosialhjelp.soknad.web.sikkerhet.Tilgangskontroll;

@Configuration
public class SoknadinnsendingLocalConfig {

    @Bean
    public Tilgangskontroll tilgangskontroll() {
        return Mockito.mock(Tilgangskontroll.class);
    }
}

File: src/test/kotlin/no/nav/sosialhjelp/soknad/config/SoknadinnsendingLocalConfig.kt
package no.nav.sosialhjelp.soknad.config

import io.mockk.mockk
import no.nav.sosialhjelp.soknad.tilgangskontroll.Tilgangskontroll
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class SoknadinnsendingLocalConfig {

    @Bean
    open fun tilgangskontroll(): Tilgangskontroll {
        return mockk(relaxed = true)
    }
}
